sdk/tables/azure-tables/review/tables.api.review.md

```diff
@@ -0,0 +1,695 @@
+  requestId?: string;
+  timeout?: number;
+}
+
+// @public
+export type GetAccessPolicyResponse = TableGetAccessPolicyHeaders &
+  SignedIdentifier[] & {
+    _response: coreHttp.HttpResponse & {
+      bodyAsText: string;
+      parsedBody: SignedIdentifier[];
+      parsedHeaders: TableGetAccessPolicyHeaders;
+    };
+  };
+
+// @public
+export interface GetEntityOptions extends coreHttp.OperationOptions {
+  queryOptions?: GeneratedQueryOptions;
+  requestId?: string;
+  timeout?: number;
+}
+
+// @public
+export type GetEntityResponse<T> = TableQueryEntitiesWithPartitionAndRowKeyResponse & {
+  value?: T;
+};
+
+// @public
+export interface GetPropertiesOptions extends coreHttp.OperationOptions {
+  requestId?: string;
+  timeout?: number;
+}
+
+// @public
+export type GetPropertiesResponse = ServiceGetPropertiesHeaders &
+  ServiceProperties & {
+    _response: coreHttp.HttpResponse & {
+      bodyAsText: string;
+      parsedBody: ServiceProperties;
+      parsedHeaders: ServiceGetPropertiesHeaders;
+    };
+  };
+
+// @public
+export interface GetStatisticsOptions extends coreHttp.OperationOptions {
+  requestId?: string;
+  timeout?: number;
+}
+
+// @public
+export type GetStatisticsResponse = ServiceGetStatisticsHeaders &
+  TableServiceStats & {
+    _response: coreHttp.HttpResponse & {
+      bodyAsText: string;
+      parsedBody: TableServiceStats;
+      parsedHeaders: ServiceGetStatisticsHeaders;
+    };
+  };
+
+// @public
+export type ListEntitiesIterator<T extends object> = PagedAsyncIterableIterator<
+  T,
+  ListEntitiesPageResult<T>,
+  {}
+>;
+
+// @public
+export type ListEntitiesOptions = Omit<TableQueryEntitiesOptionalParams, "queryOptions">;
+
+// @public
+export interface ListEntitiesPageResult<T extends object> {
+  continuationToken: TableEntityContinuationToken;
+  readonly value: T[];
+}
+
+// @public
+export type ListEntitiesResponse<T extends object> = Omit<TableQueryEntitiesResponse, "value"> & {
+  readonly value: ListEntitiesIterator<T>;
+};
+
+// @public
+export type ListTablesOptions = Omit<TableQueryOptionalParams, "queryOptions">;
+
+// @public
+export type ListTablesResponse = TableQueryHeaders &
+  TableQueryResponse & {
+    _response: coreHttp.HttpResponse & {
+      bodyAsText: string;
+      parsedBody: TableQueryResponse;
+      parsedHeaders: TableQueryHeaders;
+    };
+  };
+
+// @public
+export interface Logging {
+  delete: boolean;
+  read: boolean;
+  retentionPolicy: RetentionPolicy;
+  version: string;
+  write: boolean;
+}
+
+// @public
+export type MergeEntityResponse = TableMergeEntityHeaders & {
+  _response: coreHttp.HttpResponse & {
+    parsedHeaders: TableMergeEntityHeaders;
+  };
+};
+
+// @public
+export interface Metrics {
+  enabled: boolean;
+  includeAPIs?: boolean;
+  retentionPolicy?: RetentionPolicy;
+  version?: string;
+}
+
+// @public
+export function odata(strings: TemplateStringsArray, ...values: unknown[]): string;
```

Comment:
It seems somewhat rare to have an exported function with some associated class. How would the user use this - just curious how it ties in.

Reply:
This helper function will be used as a tagged template literal for escaping any single quotes inside a filter parameter when querying.

Reply:
That is pretty neat!